terms/roccurves_pb.tex

```diff
@@ -0,0 +1,88 @@
+
+\begin{equation}\label{eq:partial-auc}
+AUC(t_{0},t_{1}) = \int_{t_{0}}^{t_{1}} ROC(t)  dt
+\end{equation}
+
+Selecting the interval is an important practical issue but it is beyond the scope of this essay to delve further.
+
+\section{Applications of ROC}
+ROC curves have been used by numerous authors to evaluate models and scoring systems developed for pest risk assessment \citep{copp2009screening,heidy2010minfneginvspec}. They conclude that, in comparison with other models, the strength of the ROC analysis is that it thoroughly investigates test accuracy\index{roc-curves!test accuracy} across a range of scores, and that it allows for visual examination of scores on one curve or a comparison of two or more curves using a similar metric. This allows analysts to easily determine which decision threshold is most preferred, based on the desired trade-off between sensitivity and specificity, or to establish which model has the best accuracy based on the largest AUC \citep{linden2006diagpreddismgmt}. In a later paper Makowski and Mittinty \citep{makowski2010scorsysinvpests} present a procedure based on Monte Carlo simulations and ROC analysis to assess and compare scoring systems for invasive species. In their methodology the process model is a stochastic model simulating the invasion process, the candidate models are the scoring systems and the metric is the area under the ROC curve.
+
+An interesting recent paper by Fung et al. \citep{fung2018cnnbcancerclass} evidences how ROC curves statistics are used to evaluate Convolutional Neural Network for breast cancer classification. It compares the ROC curves generated from the classifier with other reviewed studies. The analysis of AUC helped in evaluating the superiority of the neural network to the compared methods based on experimental results conducted on real patient subjects.
+
+Another field where ROC proves to be contributory is psychology. Correct identification of criminals by eyewitness can help to remove a dangerous criminal from society but a false identification can lead to the erroneous conviction of an innocent suspect. A 2014 paper by Gronlund et al. \citep{gronlund2014eyewitness} describes how constructing ROC curves researchers can trace out discriminability across levels of response bias for each lineup procedure. They illustrate the shortcomings of ratio-based measures and demonstrate why ROC analysis is required for evaluating the performance of a lineup procedure. In a more recent paper Luby \citep{luby2017lineup} goes a step further and examines the application of ROC methodology to lineup data and shows that by using a log-linear analysis in conjunction with an ROC approach to eyewitness identification it is possible not only to visualise the trade-off between true positives and false positives, but also to identify which variables are interacting with one another to explain the trade-off.
```

Comment:
You have focused a lot on biomedical ... but this is not the only area where ROC are used.